### PR TITLE
UI Changes, Tab Controller

### PR DESCRIPTION
**Pull request by jchen**
Changes: 
* Contained TableView into another view with hard edges and shadow. 
* Added space on main screen for additional views/displays ("You have 4 items left!"). 
* Changed displays to a tab-bar with animated hiding/showing. 
* Changed size and display of add new item button.

### CODE REVIEW

File: BLT/BLT.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -524,12 +524,13 @@
...
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = QEJW2NUXDQ;
 				INFOPLIST_FILE = BLT/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = org.Hotchkiss.BLT;
+				PRODUCT_BUNDLE_IDENTIFIER = io.jiahua.BLT;
```
Comment (EKKOING) [inline BLT/BLT.xcodeproj/project.pbxproj]: Hmmmm... idk about that identifier

File: BLT/BLT.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -542,12 +543,13 @@
...
```
Comment (EKKOING) [inline BLT/BLT.xcodeproj/project.pbxproj]: Hmmmm... idk about that identifier

**Review by EKKOING — CHANGES_REQUESTED** [review]
The identifier needs to be changed

**Review by EKKOING — CHANGES_REQUESTED** [review]
Add Assignment Button Doesn't Show in Simulator, but does function when clicked

File: BLT/BLT.xcodeproj/project.pbxproj (hunk 1 above)
Comment (jchen) [reply BLT/BLT.xcodeproj/project.pbxproj]: Yeah I couldn't generate a provisioning profile that's why I changed it lmao.

**Pull request closed without merging**